src/main.js

```diff
@@ -25,7 +26,15 @@ const result = {
 	icon: 'icon-guests',
 	displayName: t('guests', 'Invite guest'),
 	handler: async self => {
-		return guestForm.populate(self.fileInfo, self.query)
+		const user = self.suggestions.find(s => s.isNoUser === false)
+		return new Promise((resolve, reject) => {
+			if (user) {
+				showError(t('guests', 'A similar user already exists'))
+				reject(new Error('Impossible to create guest, a similar user already exists'))
```

Comment:
Similar user sounds a bit odd, maybe we could just check if s.shareType === 0 and s.shareWith === self.query and then show something like "a user with this email address already exists" ?

Reply:
Ah though the email is the user id in that case so that might also be a bit confusing.

Reply:
Yeah the "invite guest" suggestion should be avoided when a user suggestion's email **or** user ID matches the query.
Problem is we don't have access to the future suggestion list when the condition is evaluated.

Do you think avoiding the guest creation modal is enough or should we dig a little bit to even avoid showing the guest suggestion?

Reply:
Nah, I think the current way is fine for now :+1: